src/components/theTimer.js

```diff
@@ -1,45 +1,81 @@
+
+    const children = ({ remainingTime }) => {
+        const minutes = Math.floor(remainingTime / 60)
+        const seconds = remainingTime % 60
+
+        return <div className="remaining-time"> {minutes}:{seconds}</div>
+    }
+
+    const toggleClick = () => {
+        setRunning(!running);
+        playPause.play();
+    }
 
     return (
-        <div>
-            <CountdownCircleTimer
-                key={currentindex}
-                isPlaying={running}
-                duration={timer}
-                colors={['#004777', '#F7B801', '#A30000', '#A30000']}
-                colorsTime={[7, 5, 2, 0]}
-                strokeLinecap="square"
-                onComplete={() => {
-                    setCurrentIndex(currentindex + 1);
-                    setRunning(false);
-                }
-                }
-            >
-                {({ remainingTime }) => remainingTime}
-            </CountdownCircleTimer>
-
-            <button onClick={() => { setRunning((running) => !running) }}> {running ? "PAUSE" : "START"} </button>
-            <button onClick={() => { setCurrentIndex(currentindex + 1) }}> STOP </button>
+        <div className="timer-container">
+            <div className="timer-section">
+                <div>
+                    <button className={type === "pomodoro" ? "btn-type selected-type" : "btn-type"} onClick={() => { setType("pomodoro") }}> Pomodoro </button>
+                    <button className={type === "shortbreak" ? "btn-type selected-type" : "btn-type"} onClick={() => { setType("shortbreak"); setCurrentIndex(currentindex + 1); setRunning(false); }}> Short break </button>
+                    <button className={type === "longbreak" ? "btn-type selected-type" : "btn-type"} onClick={() => { setType("longbreak"); setCurrentIndex(currentindex + 1); setRunning(false); }}> Long break </button>
+                </div>
+                <CountdownCircleTimer
+                    class="circle-timer"
+                    key={currentindex}
+                    isPlaying={running}
+                    duration={timer}
+                    colors={['#2dad26', '#dbca0d', '#db710d', '#a3130b']}
+                    colorsTime={[8, 5, 4, 0]}
+                    strokeLinecap="round"
+                    onComplete={() => {
+                        setCurrentIndex(currentindex + 1);
+                        setRunning(false);
+                    }
+                    }
+                >
+                    {children}
+                </CountdownCircleTimer>
+                <div className="btn-div">
+                    {
+                        running ? <i className="fa-solid fa-pause icon pause" onClick={toggleClick}></i>
+                            : <i className="fa-solid fa-play icon" onClick={toggleClick}></i>
+                    }
+
+                    <i onClick={() => { setCurrentIndex(currentindex + 1); setRunning(false); Stop.play() }} className="fa-solid fa-square icon stop"></i>
```

Comment:
dont put `onClick` directly on `<i>` tag. instead wrap it in a `span` and put `onClick` in it

Reply:
done in [82c9913](https://github.com/shivanigangadharan/pomodoro/pull/1/commits/82c9913194c0673d22e6e4c5971eac1e643f763f)